docs/pywbemclicommands.rst

```diff
@@ -849,7 +849,7 @@ used to create a connection and is largely the same information as is in the
 options for pywbemcli. The data includes:
 
 * **name** name of the connection (required).
-* **server_url** the url for the defined connection (required unless
+* **server** the url for the defined connection (required unless
   ``--mock-server``/``-m`` defined).
 * **default_namespace** the default namespace defined for the connection
```

Comment:
Here is a left-over use of underscore in `default_namespace` that should be hyphen.

Reply:
DONE in latest update